guides/common/modules/proc_changing-the-repository-sets-status-in-project.adoc

```diff
@@ -1,18 +1,22 @@
 [id="Changing_the_Repository_Sets_Status_in_{project-context}_{context}"]
 = Changing the Repository Sets Status in {Project}
 
-Repository sets show available repositories to each host.
-If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host.
+Repository sets show repositories available to each host.
+A host will be able to access content from a given repository if that repository is enabled.
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select a host.
-. Select the *Repository sets* tab.
+. Select the *Content* tab, then select the *Repository sets* subtab.
 On the tab, there is a set of repositories available to each host with a status of *Enabled* or *Disabled*.
 . You can override the default status by using the action menus on each table row by changing the status to *Override to disabled*, *Override to enabled*, or *Reset to default*.
+. You can also bulk select the checkboxes on each table row and use the action menu at the top.
 . On the *Overview* tab, click *Content view details* to view the environment for the host.
 +
 [NOTE]
 ====
-For hosts not in a default status or a lifecycle environment, the *Repository sets* tab shows only repositories that are relevant to the host.
+For hosts not in the default Content View and lifecycle environment, the *Repository sets* tab shows a toggle group with two options, *Limit to environment* and *Show all*.
+
+Limit to environment: Show only repositories that are relevant to the host.
+Show all: Show all available repositories including those that may not be in the host's Content View and lifecycle environment.
```

Comment:
I'm not that familiar with ASCIIdoc formatting, but I feel like this might need some. Otherwise it will just be a regular prose paragraph, right?